Designaroni/designaroni-next
# Test to master

![Replace me with a new gif](https://media.giphy.com/media/0YUPDWMaHYZnIEW6Uo/giphy-downsized.gif)

## Description

[//]: # 'Replace "JIRA-ISSUE" in the link below with the Jira Task ex: DN-000'

[JIRA ticket DN-8](https://designaroni.atlassian.net/browse/DN-8)

[JIRA ticket DN-109](https://designaroni.atlassian.net/browse/DN-109)

[//]: # "What changes are made in the pull request? minor, major & chore or cleanup tasks..."

Adds semantic release to master
Adds fix to PR template
Adds fix for sorting GraphQL top level page requests

## Documentation

[//]: # "Provide links to in-house documentation for this work or any third party documentation used"

## Impacted areas of code & app functionality

[//]: # "Describe impacted areas and areas affected by this change."

## How to recreate & verify

[//]: # "Nuerotically list steps needed to recreate the issue. Seperately list steps to verify the fix or verify that all require changes satisfy the ticket requirements."

**Steps to recreate:**

NA

**Steps to verify:**
Testing DN-8
- Once work is merged in to master, watch the github workflow to ensure it succeeds

Testing DN-109
- Once work is merged in navigate to home page https://designaroni.com
- note the top level page order in the header and footer navigation's are correct (header navigation: Home, Trips, Journal, Builds, About & footer navigation: Trips, Journal, Builds, About, Work)
- note the order of featured pages is correct (Trips, Journal, Builds, Work)

## Checklists

### Pull Request Author

I have verified...


### Code Reviewer:

I have verified...
